local_libs/lib-dex-commons/src/test/kotlin/gov/cdc/dex/azure/EventHubSenderTest.kt

```diff
@@ -10,7 +10,7 @@ class EventHubSenderTest {
 
     @Test
     fun testSendEmptyList() {
-        val ehConnStr = System.getenv("EVENT_HUB_CONNECT_STR")
+        val ehConnStr = System.getenv("EVENTHUB_CONNECTIONSTRING")
```

Comment:
if gradle defined the variable as "EVENT_HUB_CONNECT_STR", then that's the variable we should use on unit tests.
"EVENTHUB_CONNECTIONSTRING" will be at the system level.